TESTS/mbed_drivers/timeout/main.cpp

```diff
@@ -55,7 +55,7 @@ Case cases[] = {
 };
 
 utest::v1::status_t greentea_test_setup(const size_t number_of_cases) {
-    GREENTEA_SETUP(total_ticks + 5, "timing_drift_auto");
+    GREENTEA_SETUP(total_ticks + 5, "default_auto");
```

Comment:
This is an incorrect change, the host test should be set to `timing_drift_auto`. Please revert this.